WorkflowUI+SwiftUI.podspec

```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+    s.name         = 'WorkflowUI+SwiftUI'
```

Comment:
NIT: I'd personally remove the `+` in the name and file system paths. In the Xcode target name, this will translate to `_`, and will lead to some weird inconsistencies that seem unnecessary if you name this something like `WorkflowUISwiftUI`, `WorkflowUIForSwiftUI` or some other joing. Otherwise, `import WorkflowUI_SwiftUI` isn't very pretty.

Reply:
@JustinDSN and I discussed this comment. He pointed out that `WorkflowUI+SwiftUI` does not depend on `WorkflowUI` at all. So this is incorrectly named, renaming to `WorkflowSwiftUI` to indicate that this is a sibling to `WorkflowUI` and not extending it in anyway.